kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/SchemaRegistryService.java

```diff
@@ -318,14 +333,15 @@ private boolean isUnrecognizedFieldSchemaTypeMessage(String errorMessage) {
     return errorMessage.contains(UNRECOGNIZED_FIELD_SCHEMA_TYPE);
   }
 
-  private WebClient.RequestBodySpec configuredWebClient(KafkaCluster cluster, HttpMethod method, String uri) {
-    return configuredWebClient(cluster, method, uri, Collections.emptyList(),
+  private WebClient.RequestBodySpec configuredWebClient(KafkaCluster cluster) {
+    return configuredWebClient(cluster, HttpMethod.GET, SchemaRegistryService.URL_SUBJECTS, Collections.emptyList(),
```

Comment:
@jonasgeiregat 
Please do not use different URLs for the similarly named methods
With these changes, in the current class once I call `configuredWebClient` method there's gonna be a different URL used (URL_SUBJECTS/URL_SUBJECT_BY_VERSION/URL_SUBJECT_VERSIONS) based on the arguments provided.